fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -18,3316 +18,1602 @@
  * limitations under the License.
  */
 
-#include "flow/ActorCollection.h"
```

Comment:
This diff seems unrelated to a storage server component test. Or am I missing something?

Reply:
`TagPartitionedLogSystem` was not exposed as header before, it worked by implementing `ILogSystem::fromLogSystemConfig` inside `TagPartitionedLogSystem.actor.cpp` (https://github.com/apple/foundationdb/pull/4734/files/69df80d3838b79f4bd836703ffd65db6623a9a11#diff-8dbaebce2afe332a5b5e8fd15593662cc1f1c6ab31a18411cff06fa415f4dc6fL3302). It seems that we are not able to do it now after we have two types `LogSystem`.

It's annoying to have such a large change. It'll be nice if we have a way to walk around it.

Reply:
Never mind, I've figured out a way to avoid this.